test/fast/Unit tests/nvm_supports_xz

```diff
@@ -0,0 +1,43 @@
+#!/bin/sh
+
+cleanup () {
+  rm ../../xz-test/{which,command,awk,rm}
+  rmdir ../../xz-test
+}
+die () { echo $@ ; exit 1; }
+
+. ../../../nvm.sh
+
+mkdir ../../xz-test
+touch ../../xz-test/xz
+chmod +x ../../xz-test/xz
+ln -s /usr/bin/which ../../xz-test/which
+ln -s /usr/bin/command ../../xz-test/command
+ln -s /usr/bin/awk ../../xz-test/awk
```

Comment:
Please don't do this - this modifies the global system as a result of tests. It should be sufficient to create an alias, or a function, or remove `/usr/bin` from the PATH.

Tests must never modify anything on the fs outside of the test dir.